src/utils.py

```diff
@@ -18,7 +18,8 @@
 class HPOSupportedTypes:
     DIRECTIONS_SUPPORTED = ("minimize", "maximize")
     VALUE_TYPES_SUPPORTED = ("double", "int", float)
-    ALGOS_SUPPORTED = ("optuna_tpe", "optuna_tpe_multivariate", "optuna_skopt")
+    OPTUNA_ALGOS = ("optuna_tpe", "optuna_tpe_multivariate", "optuna_skopt")
+    ALGOS_SUPPORTED = (OPTUNA_ALGOS)
     TRIAL_RESULT_STATUS = ("success", "failure", "error", "prune")
```

Comment:
@khansaad - Shouldn't we be removing prune here and internally pass prune to optuna when result status is failure?